src/java.base/windows/native/libjli/java_md.c

```diff
@@ -222,6 +228,12 @@ LoadMSVCRT()
     char crtpath[MAXPATHLEN];
 
     if (!loaded) {
+        if (JLI_IsStaticallyLinked()) {
+          // For statically linked builds, we rely on the system msvcrt dlls
+          loaded = 1;
```

Comment:
Well that's a limitation.

Reply:
Yes, maybe. A quick googling gives that it appears possible, but not recommended, to statically link msvcrt libraries, so I don't think we should try to go down that route. 

However, the situation is not principally different from other situations where we rely on system libraries. I know that historically msvcrt libraries of incorrect versions (or entirely missing) have been more of an issue, but I wonder how much so it is in the present. I don't think we'll know for sure until this has been tested "in the wild".